ts/torch_handler/base_handler.py

```diff
@@ -138,6 +171,21 @@ def initialize(self, context):
         else:
             raise RuntimeError("No model weights could be loaded")
 
+        self.model.eval()
+        optimization_config = os.path.join(model_dir, "compile.json")
+        backend = load_compiler_config(optimization_config)
+
+        # PT 2.0 support is opt in
+        if check_pt2_enabled() and backend:
+            # Compilation will delay your model initialization
+            try:
+                self.model = torch.compile(self.model, backend=backend)
```

Comment:
QQ, we are assuring the numerical stability so user wouldn't expect any accuracy drop/ difference?

Reply:
Good question, in general we do guarantee it but there will be small numerical differences but we have tools to debug https://pytorch.org/docs/master/dynamo/troubleshooting.html#accuracy-debugging